target_chains/starknet/contracts/src/wormhole.cairo

```diff
@@ -327,4 +377,22 @@ mod wormhole {
         hasher.push_u256(y);
         hasher.finalize() % ONE_SHIFT_160
     }
+
+    #[generate_trait]
+    impl PrivateImpl of PrivateImplTrait {
+        fn verify_governance_vm(self: @ContractState, vm: @VM) {
```

Comment:
This is very nitpicky but as far as I can tell from the current code, `VM` is only ever produced by parse and verify, might be nice to rename it something like `VeririedVM` because it implies that you only ever have a VM you can work with if you have passed verification which makes code like you've added now clearer in that there won't be a time where governance might have an unverified VM.

Reply:
I agree. Renamed.